atlas/getting-started/deploy-from-existing-repo.mdx

```diff
@@ -0,0 +1,112 @@
+
+1. Fill out a name for your application. We recommend choosing a name similar to your GitHub repository's name. The application name cannot be changed once it has been deployed.
+
+2. Select a region for your application. The region you select will be used for all of your app's environments (production/staging/etc.) and cannot be changed once your Atlas application has been deployed.
+
+3. Click `Continue`.
+
+![Choose a name and region](./assets/img/wp-engine-settings.png)
+
+### 5. Configure the environment
+
+1. From the list of your repository's branches, select the branch you want to base your environment on. Whenever changes are pushed to this branch, your app will be automatically re-deployed.
+
+Examples:
+
+- If you want to create a production environment, you could use your repo's `main` branch
+- If you want to create a staging environment, you could use your repository's `staging` branch
+
+![Select a branch](./assets/img/select-branch.png)
+
+2. Enter a name for your Atlas environment. The name should represent how you intend to use the environment. For example, enter something like `prod` if it's a production environment, `staging` if it's a staging environment, etc.
+
+3. Every headless environment is associated with a WordPress environment. Choose either:
+
+   - `I already have a WordPress environment`: Use a WordPress site from your WP Engine account. Select the search bar and search for the name of your WordPress site.
+   - `I want to create a new WordPress environment`: WP Engine makes a new WordPress site and links it to your application. In the box, enter a name for your new WordPress site.
+
+![Link to WordPress](./assets/img/link-to-wordpress.png)
+
+4. In order for your WordPress environment to become "headless", Atlas installs the [WP GraphQL](https://wordpress.org/plugins/wp-graphql/) and [Atlas Content Modeler](https://wordpress.org/plugins/atlas-content-modeler/) plugins.
+
+If you're using the Faust.js Starter Kit or an existing Faust.js app repository, check the `I am using Faust.js` checkbox. This will ensure that the [FaustWP](https://wordpress.org/plugins/faustwp/) plugin is also installed.
+
+![Select Faust.js Plugin](./assets/img/faustjs.png)
+
+**NOTE:** If you're using an existing application (rather than the Faust.js Starter Kit), see the [Environment variables (Optional)](#environment-variables--optional-) section below to set up environment variables before proceeding with the steps below.
+
+5. Select `Create App`.
+
+6. The Atlas app details page appears. When Atlas completes building your application, a checkmark appears under `Last Build`.
+
+![Headless App Building](./assets/img/portal-app-building.png)
+
+7. To visit your headless WordPress app once the first deploy finishes, click the link under the `Atlas URL` heading. Your Atlas app opens in a new browser tab. Enjoy!
+
+![First app deployed](./assets/img/first-app-deployed.png)
+
+### Set Environment Variables (Optional)
```

Comment:
I like the use of the optional anchor link here to keep them in the deployment flow